requirements-UBUNTU.txt

```diff
@@ -1 +1,2 @@
 jsonschema
+cheetah3
```

Comment:
Please, move it to requirements-travis.txt or create a separated file for github actions. I'd like to avoid getting this into the wide used requirements-UBUNTU.txt because it is going to install the latest version and I am afraid it can lead us in some issues.

Reply:
done